Sg cleanup

saiemgilani opened the pull request:
**Update 01-Data-Ingest.R for formatting primarily** 
1) move user-defined variables to top of script.
2) change epa_wpa variable explicitly to false, uncommenting sys.sleep
3) rename td_e to td_vec for consistency
4) move penalty detection to function in 06-Data-Ingest-Utils.R
5) mainly just formatting/fixing comments

**Update 02-EPA-Model.R** 
1) Remove penalties from remove_play vector
2) commenting out non fg ep_model training
3) add slight ggplot2 styling to EPA histograms for training years.
4) more plot styling

**Update 06-Data-Ingest-Utils.R** 
as mentioned in 01-Data-Ingest.R comments, add_penalty_cols function added to 06-Data-Ingest-Utils.R

**Update 07-CV-Utils.R** 
remove extra mutation on NSH and Next_Score after prediction for preds_ep df since with new tidyverse update causes duplicate columns under default setting.

**Update 08-Pred-Utils.R**
1) add a couple idiosyncratic play_types to turnover play type
2) rearranging some variables cause I'm ocd
3)  some vector variables renamed for consistency.
4) also now gives pre-join rows and post-join rows for clear duplicate row detection earlier during query runs

Update 03-WPA-Model.R 
updating save paths
wpa plotting updating
trying to get these working, updated data path

saiemgilani committed:
Update 01-Data-Ingest.R for formatting primarily

1) move user-defined variables to top of script.
2) change epa_wpa variable explicitly to false, uncommenting sys.sleep
3) rename td_e to td_vec for consistency
4) move penalty detection to function in 06-Data-Ingest-Utils.R
5) mainly just formatting/fixing comments

saiemgilani committed:
Update 02-EPA-Model.R

1) Remove penalties from remove_play vector
2) commenting out non fg ep_model training
3) add slight ggplot2 styling to EPA histograms for training years.

saiemgilani committed:
Update 06-Data-Ingest-Utils.R

as mentioned in 01-Data-Ingest.R comments, add_penalty_cols function added to 06-Data-Ingest-Utils.R

saiemgilani committed:
Update 07-CV-Utils.R

remove extra mutation on NSH and Next_Score after prediction for preds_ep df since with new tidyverse update causes duplicate columns under default setting.

saiemgilani committed:
Update 08-Pred-Utils.R

1) add a couple idiosyncratic play_types to turnover play type
2) rearranging some variables cause I'm ocd
3)  some vector variables renamed for consistency.

saiemgilani committed:
Update 02-EPA-Model.R

more plot styling

saiemgilani committed:
Update 08-Pred-Utils.R

also now gives pre-join rows and post-join rows for clear duplicate row detection earlier during query runs

saiemgilani committed:
Update 03-WPA-Model.R

updating save paths

saiemgilani committed:
wpa plotting updating

trying to get these working, updated data path

saiemgilani committed:
Update 02-EPA-Model.R

no actual changes here

saiemgilani committed:
Update 03-WPA-Model.R

update calibration error value

saiemgilani committed:
add calibration plots

plots added to new folder: figures

saiemgilani committed:
Update 02-EPA-Model.R

* add home_offense variable
* style calibration plots

saiemgilani committed:
Update 08-Pred-Utils.R

* add home_offense variable and home_offense_end variables

meysubb commented:
Okay so I'm going to put penalties back into the removal process.

meysubb commented:
Secondly need to make sure the post-prediction steps are added into the package so that we have consistency.

meysubb committed:
remove penalties

meysubb commented:
In 08_Pred_Utils I think we should stop relying on the function that is written here and we should flip over to using the function in cfbscrapR. 

So that way we can maintain consistency. 

No longer doing this, because we want to test things here then move to the package.

saiemgilani committed:
Update 01-Data-Ingest.R

added function pbp_full_counts_df - creating a few binary and running total columns for pass, rush, kickoffs, punts, field goals for each team's offense and defense. Separate rollup for drive, in addition to the full running total for each play type.

saiemgilani committed:
Update 06-Data-Ingest-Utils.R

added function pbp_full_counts_df - creating a few binary and running total columns for pass, rush, kickoffs, punts, field goals for each team's offense and defense. Separate rollup for drive, in addition to the full running total for each play type.

saiemgilani committed:
Update 07-CV-Utils.R

adding a little verbosity

saiemgilani committed:
Update 08-Pred-Utils.R

*fix ep_before for play_types "Defensive 2pt Conversion" to 1.
*Value of 1 indicating a neutral point after (and the like 40-47% conversion rate of 2pts)
*ep_after then set to -2.

saiemgilani committed:
Update 03-WPA-Model.R

updating some plotting to newer styling. Potential candidate features added to WPA model are home_offense and offense_epa_ratio and newer WPA model added. Not overly sold either is worth including

saiemgilani committed:
Update 02-EPA-Model.R

added penalties back to remove plays... lol. start of an idea for tdd on final epa outputs

saiemgilani committed:
add calibration plots

add calibration plots

saiemgilani committed:
Create histogram_epa_season_2019.png

saiemgilani committed:
Delete ep_fg_cv_loso_calibration_results_new.png

saiemgilani committed:
Merge branch 'sg_cleanup' into ep_wp_hfa

saiemgilani committed:
Merge pull request #17 from meysubb/ep_wp_hfa

Ep wp hfa
